tests/checkpoint/test_checkpoint.py

```diff
@@ -1450,11 +1467,309 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_with_
     assert validation_result.meta["checkpoint_name"] == checkpoint_name
 
 
+@pytest.mark.integration
+@pytest.mark.slow  # 1.15s
+def test_newstyle_checkpoint_raises_error_if_batch_request_and_validator_are_specified_in_constructor(
+    titanic_pandas_data_context_with_v013_datasource_with_checkpoints_v1_with_empty_store_stats_enabled,
+    batch_request_as_dict,
+):
+    context: FileDataContext = titanic_pandas_data_context_with_v013_datasource_with_checkpoints_v1_with_empty_store_stats_enabled
+    batch_request: BatchRequest = BatchRequest(**batch_request_as_dict)
+    context.add_expectation_suite("my_expectation_suite")
+    validator: Validator = context.get_validator(
```

Comment:
Do we need a real expectation suite here? Can we mock the context?

Reply:
@anthonyburdi This pattern is used throughout, and for now -- consequently -- the tests are marked as integration; I do agree that (as a separate effort) -- these tests can be lightened up through the use of mocks.

Reply:
@anthonyburdi Done -- thanks!